### DISCUSSION TITLE
Prevent changing default page

### DISCUSSION THREAD

**Discussion by AndiLeni** [joomla/joomla-cms#32151] [Q&A]
Hello,

I have the problem, that some editors on my site are changing the default site in the menus by accident.
Is there a way to prevent this? F.e. a setting to prevent a group fom changing the default page?

Kind regards
Andreas

**Comment by brianteeman** [comment]
They have to do a lot for it to be by accident.

There is no setting you can use

**Reply by AndiLeni** [reply]
What do you mean with "a lot"?
It's just clicking the star-button next to a menu item to make this happen. 

Is there maybe another way to prevent this?

**Comment by ghost** [comment]
Have you asked at the Forum?

**Comment by HLeithner** [comment]
@AndiLeni core has no accesslevel for this and adding an access level for setting the default menu items doesn't make sense at it's own there would be a bigger (structured) acl rework to implement this.

I would suggest to find/write (or someone who create it for you depending on your coding skills) a plugin that intercepts the store request and checks if the use is allowed to do it.

**Reply by AndiLeni** [reply]
Thanks for the answer!
It is a pity that this is not possible by default.

Do you know which plugin event would be needed to check?
